### PR TITLE
session, metrics: refine the resource control metrics

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Issue Number: ref #38825.

Problem Summary:

### What is changed and how it works?

- `tidb_executor_statement_total` doesn't have the resource group label, so it's not accurate to calculate the RU/QPS by using it. This PR introduces a new metric named `tidb_session_resource_group_query_total`.
- Use the log-based Y axe.
- Refine the PromQL.

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [x] Manual test (add detailed scripts or steps below)

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]
lgtm

File: metrics/session.go [hunk 1]
```diff
@@ -178,45 +178,54 @@ var (
...
+	LblRunning           = "executing_sql"
+	LblLockWaiting       = "waiting_for_lock"
+	LblCommitting        = "committing"
+	LblRollingBack       = "rolling_back"
+	LblHasLock           = "has_lock"
+	LblPhase             = "phase"
+	LblModule            = "module"
+	LblRCReadCheckTS     = "read_check"
+	LblRCWriteCheckTS    = "write_check"
+	LblResourceGroupName = "name"
```
Comment (glorv) [inline metrics/session.go:230]:
```suggestion
	LblName = "name"
```
Other "name" labels can share the same const

File: session/session.go [hunk 2]
```diff
@@ -2215,6 +2215,15 @@ func (s *session) ExecuteStmt(ctx context.Context, stmtNode ast.StmtNode) (sqlex
 		recordSet, err = runStmt(ctx, s, stmt)
 	}
 
+	// Observe the resource group query total counter if the resource control is enabled and the
+	// current session is attached with a resource group.
+	if variable.EnableResourceControl.Load() && !s.isInternal() {
```
Comment (glorv) [inline session/session.go:2220]: What about only check `len(resourceGroupName) != 0` here which means the resource group is explicitly set by user so we can support explicitly set resource group to the default group, and maybe at some time the internal session should also bind to a group that should be tracked.

**Review by glorv — COMMENTED** [review]
I see in the grafana page we can select "instance" labels for tidb instances, but none of the charts in this page use this label, and most of the expressions are from pd metrics, so I suggest remove this label selector.

**Review by glorv — APPROVED** [review]

**Comment by JmPotato** [comment]
@bb7133 Could you please help leave an approve for this PR? Thx.

**Review by tiancaiamao — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]

**Pull request merged**